Mathlib/Control/Applicative.lean

```diff
@@ -35,7 +35,7 @@ theorem Applicative.map_seq_map (f : α → β → γ) (g : σ → β) (x : F α
   simp [flip, functor_norm]
 #align applicative.map_seq_map Applicative.map_seq_map
 
-theorem Applicative.pure_seq_eq_map' (f : α → β) : (· <*> ·) (pure f : F (α → β)) = (· <$> ·) f :=
+theorem Applicative.pure_seq_eq_map' (f : α → β) : (· <*> ·) (pure f : F (α → β)) = (f <$> ·) :=
```

Comment:
Presumably this should be something like:
```suggestion
theorem Applicative.pure_seq_eq_map' (f : α → β) : (pure f : F (α → β) <*> ·)  = (f <$> ·) :=
```
though it would be worth checking how Lean3 wrote it